src/java.base/share/native/libjli/wildcard.c

```diff
@@ -100,14 +100,6 @@
 #include <dirent.h>
 #endif /* Unix */
 
-#if defined(_AIX)
-  #define DIR DIR64
-  #define dirent dirent64
-  #define opendir opendir64
-  #define readdir readdir64
-  #define closedir closedir64
-#endif
```

Comment:
You have to keep these defines for AIX, because _LARGE_FILES does not act on them, but instead of 
`#if defined(_AIX)`
you could use 
`#if defined(_AIX) && defined(_LARGE_FILES)`